src/gui/app.cpp

```diff
@@ -1,18 +1,150 @@
 #include "app.h"
 
+#include "arguments.h"
+#include "mainwindow.h"
+#include "printinfo.h"
+#include "version.h"
+
 #include <QDir>
 #include <QFileInfo>
 #include <QFileOpenEvent>
 #include <QSettings>
 #include <QTextStream>
-
-#include "arguments.h"
-#include "mainwindow.h"
-#include "printinfo.h"
-#include "version.h"
+#include <vector>
 
 namespace NeovimQt {
 
+MainWindow* s_lastActiveWindow{ nullptr };
```

Comment:
nitpick:

Try to avoid `T*` if `nullptr` is not an expected value.

Ex) `&` for required parameters/values and `*` for optional parameters/values. 

This fix might be hard for `s_lastActiveWindow` as you've written the code... If so, NBD.

I think this can be easily applied to `s_windows`, `std::vector<MainWindow&>`